src/Mollie/WC/Payment/Order.php

```diff
@@ -1094,13 +1094,16 @@ protected function updatePaymentDataWithOrderData($order, $orderId)
     /**
      * Method that shortens the field to a certain length
      *
-     * @param $field
-     * @param int $maximalLength
+     * @param string $field
+     * @param int    $maximalLength
      *
      * @return false|string
      */
     protected function maximalFieldLengths($field, $maximalLength)
     {
+        if (!is_string($field)) {
+            return null;
```

Comment:
I suggest to return `false` here, otherwise add it to `@return` like so `false|string|null`.